Sources/BowEffects/Data/Internal/DispatchTimeInterval+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+extension DispatchTimeInterval {
+    func toDouble() -> Double? {
+        switch self {
+        case .seconds(let value): return Double(value)
+        case .milliseconds(let value): return Double(value) * 0.001
+        case .microseconds(let value): return Double(value) * 0.000001
```

Comment:
```suggestion
        case .microseconds(let value): return Double(value) * 0.000_001
```

I think it is easier to read

Reply:
same here!